### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 15

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#315] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152322953-9cacc17f-3663-4d39-8c20-dd22f69b8432.png)

**Comment by Exyss** [comment]
Credo sia abbastanza autoesplicativo, poiché una funzione biettiva F : X -> Y dove #X = #Y equivale ad effettuare delle distribuzioni semplici di N elementi

![Esercizio 15](https://user-images.githubusercontent.com/65043207/152659905-1d80ffc0-41f0-4fb6-a51b-133872052e37.jpg)

**Comment by GattoBrizzolato** [comment]
.

**Comment by LuginiAndrea** [comment]
Spiego solo il passo induttivo in quanto il caso base è banale.
Abbiamo sia $X$ che $Y$ con $n+1$ elementi:
prendiamo l'$n+1$-esimo elemento tdi $Y$, chiamiamolo $y$ rimuoviamolo da $Y$. 
Scegliamo ora un elemento $x \in X$ scelto in modo tale che $x \rightarrow y$.
Abbiamo $n+1$ modi di scegliere questo elemento.
Ci rimangono ora $X - x, Y - y$ con $n$ elementi, e sappiamo che le biezioni fra i due sono $n!$.
La risposta risulta quindi $\left(n+1\right) \times n! = \left(n+1\right)!$, ovvero i modi di scegliere $x \rightarrow y$ ed i modi di mettere in biezione i restanti elementi
